.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '10.x'
+
+    - run: npm i
```

Comment:
can you change it to npm ci instead ?

Reply:
we dont have a lockfile, so I don't think this would work, right? I suppose I could add one!

Reply:
Ok let's keep an npm i then.